.travis.yml

```diff
@@ -1,16 +1,15 @@
 language: rust
-env:
-  - BOILERPLATEAPP_DATABASE_URL=postgres://localhost/boilerplateapp
 services:
   - postgresql
 rust:
-  - nightly
+  - nightly-2017-12-21
```

Comment:
Let's document this (with link to the rocket/ring issue) for why we need this.

Reply:
I don't think we are going to need this, since the `ring` crates problem is going to be resolved after new rust version 1.23 scheduled to be released in January 4th.

https://github.com/SergioBenitez/Rocket/issues/513